bower.json

```diff
@@ -1,11 +1,14 @@
 {
   "name": "bootstrap-datepicker",
   "version": "1.4.0",
-  "main": ["js/bootstrap-datepicker.js", "css/datepicker.css", "css/datepicker3.css"],
+  "main": [
+    "dist/css/bootstrap-datepicker.css",
+    "dist/css/bootstrap-datepicker3.css",
+    "js/bootstrap-datepicker.js"
+  ],
   "license": "Apache-2.0",
   "dependencies": {
     "jquery" : ">=1.7.1",
     "bootstrap" : ">=2.0.4 <4.0"
-  },
-  "ignore": {}
```

Comment:
I've added this empty ignore tag because of #1199, so this should be preserved to avoid error's when running bower install